static/js/zamboni/devhub.js

```diff
@@ -376,6 +380,16 @@ function addonFormSubmit() {
     })(parent_div);
 }
 
+function refreshAddonBasicInformation($form) {
+    // Retrieve the div containing the updated name and then its value
+    update_name_div = $form.find('[data-name=name]')
+    updated_name = update_name_div.text()
```

Comment:
Get rid of the intermediate variable and do declare the other one. 
`var updated_name = $form.find('[data-name=name]').text()`